docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -59,23 +59,104 @@ For static onboarding, be sure to use the [Gateway static definition example](#g
 
 ## Establishing a trust relationship between domain Gateways and the Cloud Gateway
 
-The following keytool commands are examples of establishing a trust relationship between domain Gateways and the Cloud Gateway.
-
-- Import the public key certificate of all domain Gateways into the truststore of the Cloud Gateway.
-
-  Use the following example of the keytool command when the domain Gateways running on CA11 and CA32 import the certificate into the Cloud Gateway running on CA31:
-
-  `keytool -import -file keystore/ca11/local_ca/local_ca.cer -alias gateway_ca11 -keystore keystore/ca31/localhost/localhost.truststore.p12`
-
-  `keytool -import -file keystore/ca32/local_ca/local_ca.cer -alias gateway_ca32 -keystore keystore/ca31/localhost/localhost.truststore.p12`
-
-- Import a public key certificate of the Cloud Gateway into the truststore of all domain Gateways.
-
-  Use the following example of the keytool command  when the certificate of the Cloud Gateway running on CA31 is imported into the truststore of the domain Gateways running on CA11 and CA32:
-
-  `keytool -import -file keystore/ca31/local_ca/local_ca.cer -alias gateway_ca31 -keystore keystore/ca11/localhost/localhost.truststore.p12`
-
-  `keytool -import -file keystore/ca31/local_ca/local_ca.cer -alias gateway_ca31 -keystore keystore/ca32/localhost/localhost.truststore.p12`
+For routing to work in a MultiTenancy configuration, the Cloud Gateway must trust the domain Gateways, and vice versa.
+In other words the root and intermediate (if applicable) public certificates needs to be shared between Cloud Gateway and domain Gateways. 
+
```

Comment:
Can we add some image here showing the system and the pieces to make it easier to visualize the system X, Y and Z described later?

Reply:
Done